src/test/components/ButtonReset.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import { ButtonReset } from '../../forms/ButtonReset';
+
+import { CLIENT_DEFAULTS } from '../../utils/CLIENT_DEFAULTS';
+
+
+describe('Test ButtonReset component', () => {
+  it('Test click event', () => {
+    let mockCallBack = jest.fn();
+
+    let button = shallow(<ButtonReset onClick={mockCallBack} snippets={{}}></ButtonReset> )
```

Comment:
Just a note for the future - if you wrap this in a `<div>` you can then do `.find('ResetButton')` instead.